frame/composable-traits/src/vault.rs

```diff
@@ -27,6 +28,18 @@ pub enum FundsAvailability<Balance> {
 	None,
 }
 
+#[derive(Copy, Clone, Encode, Decode, MaxEncodedLen, Default, Debug, PartialEq, TypeInfo)]
+pub struct StrategyOverview<Balance> {
+	/// The allocation of this strategy.
+	pub allocation: Perquintill,
+	/// The reported balance of the strategy.
+	pub balance: Balance,
+	/// Sum of all withdrawn funds.
+	pub lifetime_withdrawn: Balance,
+	/// Sum of all deposited funds.
+	pub lifetime_deposited: Balance,
+}
+
```

Comment:
@PoisonPhang Because I created `get_strategies` function which use [`StrategyOverview`](https://github.com/ComposableFi/composable/blob/9f3b4c6e4c1ad70b58b4fd956c815c79eb2234a7/frame/composable-traits/src/vault.rs#L137-L139) in its annotation. So, I was needed to have this struct in the same file.

Reply:
Maybe my question will seem stupid, but for what reason we can't use the structure from `models.rs` file?

Reply:
Maybe my question will seem stupid, but for what reason we can't use the structure from `models.rs` file?

Reply:
In the course of my work with other pallets, I noticed that the structures used in the traits are also created from the same file. Therefore, I decided to do so in order to maintain some generality.

Reply:
@PoisonPhang Also the `vault pallet` itself has a dependency on `composable-traits`, and then it turns out there will be a cycle if I try to import the `models.rs` file into a `composable-traits`.